pkg/server/helper.go

```diff
@@ -44,15 +44,44 @@ func GeneratePTS(depBody deploymentPost, org, env string) (v1.PodTemplateSpec, e
 				TargetPath:    "/",
 			},
 		}
-		intPort = 9000
+		containerPorts = []v1.ContainerPort{
+			{
+				ContainerPort: int32(9000),
+			},
+		}
 		var err error
 		tempPaths, err = composePathsJSON(defaultPath)
 		if err != nil {
 			return v1.PodTemplateSpec{}, err
 		}
 	} else {
+		//We were given Paths
 		var err error
-		intPort, err = strconv.Atoi(depBody.Paths[0].ContainerPort)
+		//Check to see if we were given multiple ports
+		if multipleEdgePorts(depBody.Paths) {
```

Comment:
Can we combine the single and multiple cases? If we always run  the for loop the single port is still covered right?